src/runtime/composables/local/useAuth.ts

```diff
@@ -87,6 +87,28 @@ const signOut: SignOutFunc = async (signOutOptions) => {
     }
   }
 
+  // Explicitly clear both cookies to avoid watcher issues
+  // Use the same options to clear the cookie as the ones used to set it
+  useCookie(config.token.cookieName, {
+    maxAge: 0,
+    domain: config.token.cookieDomain,
+    sameSite: config.token.sameSiteAttribute,
+    secure: config.token.secureCookieAttribute,
+    httpOnly: config.token.httpOnlyCookieAttribute
+  }).value = null
```

Comment:
I've been trying to understand why is this required - could you please clarify that?

Reply:
coding is hard! for skill issues, use chatgpt